### DISCUSSION TITLE
Mac OS X traffic lights for Windows

### DISCUSSION THREAD

**Discussion by GameBoyzGB** [nuttyartist/notes#559] [Ideas]
I'm happy with this Notes app since I started using it during the early days and it has come a long way. I'm still hoping for it to really look like a real Mac OS X application using the circular red, yellow and green buttons in place of windows close, minimize and maximize icons. Would that be possible to do or is there a way to customize this on my own?
Thanks!
![image](https://user-images.githubusercontent.com/100013857/227765202-e958fb4a-0e65-473a-a7d0-a45bc02ee603.png)

**Comment by nuttyartist** [comment]
Hi! On what operating system are you running Notes? 

Do you want the macOS-style traffic lights on Windows? That would be quite odd for us to support. Why would you want that? Do you prefer their aesthetic? We're trying to be as native looking as we can on each platform, so supporting that feature is against our goal.

**Reply by GameBoyzGB** [reply]
I'm on a windows 11 desktop but have themed it to look like Mac OS because I want it to match the look and feel of my Macs and Macbooks. Right now, your notes app is the only one that looks out of place with my desktop and applications.

**Comment by GameBoyzGB** [comment]
![image](https://user-images.githubusercontent.com/100013857/229004053-e6c804be-d883-4dc0-9fe0-214895c11d7b.png)

**Reply by DeeptangshuSaha** [reply]
Hey which browser is this ?? And how did you change it to look like that??

**Reply by XXYazimmmm** [reply]
> ![image](https://user-images.githubusercontent.com/100013857/229004053-e6c804be-d883-4dc0-9fe0-214895c11d7b.png)

btw, how do u get these traffic controls on windows? i have wanted these

**Comment by zjeffer** [comment]
What happens if you go enable `View > Use native window frame` ?

**Reply by GameBoyzGB** [reply]
![image](https://user-images.githubusercontent.com/100013857/229185388-96f0f522-81f8-4aaf-873c-4324c9d08d58.png)

**Reply by GameBoyzGB** [reply]
My other applications are skinned. It would be nice if you can change skins and allow them to be user created.

**Reply by zjeffer** [reply]
Do you use any other qt-based apps that you were able to skin?

I think it's possible to supply your own QtStyleSheet to Qt applications, but I haven't looked that far into the details of it.

**Reply by nuttyartist** [reply]
How come your other applications are skinned while Notes (using native window decoration) isn't?

**Reply by GameBoyzGB** [reply]
All the applications I use are specifically made with a Mac OS interface except Firefox which is skinned to look like Safari. The Dock/Finder is a Windows app sold in Steam called MyDockFinder.  The calculator was built from scratch to closely copy the Mac version. I'm sure if I used something like Stardock's Windowblinds, it would skin your Notes but I am trying to use as less memory as possible. I'm just wondering why you made notes with the MinMaxClose buttons on the left side just like a Mac but not have it skinned as well.

**Reply by nuttyartist** [reply]
Well, the reason the buttons are on the left side is that back then I used a Windows version with a Hebrew layout - so everything was in the opposite direction (since in Hebrew we write from right to left). But the buttons were always meant to resemble Windows look as much as we can. Unfortunately, we don't expect to support macOS-style buttons on Windows. And as for the next version we'll default to native decoration, rather than our custom style.
